horovod/runner/elastic/discovery.py

```diff
@@ -15,20 +15,27 @@
 
 import io
 import logging
+import random
 import threading
-
+import time
 from collections import defaultdict
 
 from horovod.runner.common.util import safe_shell_exec
 from horovod.runner.elastic.worker import HostUpdateResult
-
-
+# Maximum time hosts are in cooldown period
+COOLDOWN_UPPER_LIMIT_SECONDS = 5 * 60
+# Minimum time hosts are in cooldown
+COOLDOWN_LOWER_LIMIT_SECONDS = 10
+# Unit of increasing cooldown for repeat failure
+COOLDOWN_PERIOD_DELAY_SECONDS = 10
```

Comment:
https://github.com/horovod/horovod/issues/1926#issuecomment-992722712

Should it be `Delta` or `Delay`?

Reply:
I removed this in light of our conversation on the `--cooldown-range`